docs/ARCHITECTURE.md

```diff
@@ -2,7 +2,7 @@
 
 This project was created using a library called [Sapper][sapper_home] which uses [Svelte][svelte_home] to create efficient, static websites.
 
-## `routes` 📁
+## `/src/routes` 📁

```

Comment:
(Nitpick) A good way to express directories vs. files in the path is to use a trailing slash: `/src/routes/`.

Reply:
Oh that is a good point. I will replace them all in a future commit.